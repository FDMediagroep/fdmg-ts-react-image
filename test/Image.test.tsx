import * as Enzyme from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import { shallow, mount } from 'enzyme';

import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Img from "../src/Image";

console.info = function() {};

beforeAll(() => {
    Enzyme.configure({ adapter: new Adapter() });
});

test('Image renders correctly', () => {
    let component = renderer.create(
        <Img alt={"alt-text"} src={'test.png'} className={'css-class-name'}/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
