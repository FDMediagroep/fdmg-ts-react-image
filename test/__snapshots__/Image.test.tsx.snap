// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image renders correctly 1`] = `
<img
  alt="alt-text"
  className="css-class-name"
  src="test.png"
/>
`;
